### PR TITLE
fix: show empty accessible

### LINKED ISSUE #700: Copy modal show error alert when accessible is empty

### PR DESCRIPTION
**Pull request by pyphilia**
fix #700

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
LGTM 🪐

File: src/components/collection/copyModal/AccessibleNavigationTree.tsx [hunk 1]
```diff
@@ -18,13 +19,15 @@ interface AccessibleNavigationTreeProps {
...
 const AccessibleNavigationTree = ({
+  isDisabled,
   onClick,
   onNavigate,
   selectedId,
 }: AccessibleNavigationTreeProps): JSX.Element => {
-  const { t } = useLibraryTranslation();
-  const [page, setPage] = useState(1);
   const { hooks } = useContext(QueryClientContext);
+  // todo: to change with real recent items (most used)
```
Comment (spaenleh) [inline src/components/collection/copyModal/AccessibleNavigationTree.tsx:28]: The comment is not at the right place no ?

File: src/components/collection/CollectionsGrid.tsx [hunk 2]
```diff
@@ -40,8 +40,8 @@ const CollectionsGrid = ({
         justifyContent="flex-start"
         id={id}
       >
-        {Array.from({ length: 4 }).map(() => (
-          <Grid xs={6} sm={4} md={3} lg={3} xl={2}>
+        {Array.from({ length: 4 }, (_, idx) => idx).map((idx) => (
```
Comment (spaenleh) [inline src/components/collection/CollectionsGrid.tsx:43]: I think in cases like this, we should pre-generate a list with fixed ids and use that to map over. Using the index completely defies the purpose of using a key in react.

**Review by spaenleh — APPROVED** [review]

File: src/components/collection/CollectionsGrid.tsx (hunk 2 above)
Comment (pyphilia) [reply src/components/collection/CollectionsGrid.tsx:43]: It's kinda the same, except I used index to fill the map.

**Pull request merged**